.gitattributes

```diff
@@ -1,4 +1,3 @@
 sql/parser/sql.go -diff
 ui/embedded.go -diff
-*.pb.go -diff
-*.pb.gw.go -diff
+*.pb.* -diff
```

Comment:
Right, I think a point that Ben made earlier in this discussion is that
these files are often merged correctly by git, so -diff is appropriate.
On Jun 20, 2016 18:54, "David Taylor" <notifications@github.com> wrote:

> In .gitattributes
> <https://github.com/cockroachdb/cockroach/pull/7344#discussion_r67781535>:
>
> > @@ -1,4 +1,3 @@
> >  sql/parser/sql.go -diff
> >  ui/embedded.go -diff
> > -*.pb.go -diff
> > -*.pb.gw.go -diff
> > +*.pb.* -diff
>
> no, -diff doesn't, it just disables patch and diff generation.
> merge is what controls merge behavior, so -merge (or merge=binary) will
> cause merges to leave the working copy but mark it conflicted.
>
> The binary macro is -diff -merge -text.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/7344/files/9c77203d0d1da8708aeddbb2a32f8a11e6e560d8#r67781535>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABdsPKfq7IY21qJW2mHZWfk3dvqxOCnYks5qNxoMgaJpZM4I6Li->
> .
>